core/src/nodes/raw_swarm.rs

```diff
@@ -192,6 +195,83 @@ where
     },
 }
 
+impl<'a, TTrans, TInEvent, TOutEvent, THandler> fmt::Debug for RawSwarmEvent<'a, TTrans, TInEvent, TOutEvent, THandler>
```

Comment:
This should have been in a different pull request, but let's not remove it now.

Reply:
Reason was I needed a `Debug` impl in order to make asserts and I didn't want to block this PR needlessly.